docs/source/installation.rst

```diff
@@ -35,4 +35,68 @@ The latest release is |release|.
 
     pip install git+https://github.com/mesosphere/dcos-e2e.git@master
 
+
+Windows
+~~~~~~~
+
+The only supported way to run DC/OS E2E on Windows is using Vagrant and VirtualBox.
+
+- Ensure Virtualization and VT-X support is enabled in your PC's BIOS. Disable Hyper-V virtualisation. See https://www.howtogeek.com/213795/how-to-enable-intel-vt-x-in-your-computers-bios-or-uefi-firmware/
+- Install `VirtualBox`_ and VirtualBox Extension Pack.
+- Install `Vagrant`_.
+- Install the Vagrant plugin for persistent disks:
+
+.. code:: ps1
+
+    vagrant plugin install vagrant-persistent-storage
+
+- Optionally install the Vagrant plugins to cache package downloads and keep guest additions updates:
+
+.. code:: ps1
+
+        vagrant plugin install vagrant-cachier
+        vagrant plugin install vagrant-vbguest
+
+- Start Powershell and download the E2E ``Vagrantfile`` to a directory containing a DC/OS installer file:
+
+.. code:: ps1
+
+    ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/mesosphere/dcos-e2e/master/vagrant/Vagrantfile')) | Set-Content -LiteralPath Vagrantfile
+
+- By default, the ``Vagrantfile`` installs DC/OS E2E from the most recent release at the time it is downloaded.  To use a different release, or any Git reference, set the environment variable ``DCOS_E2E_REF``:
+
+.. code:: ps1
+
+    $env:DCOS_E2E_REF = "master"
+
+- Start the virtual machine and login:
+
+.. code:: ps1
+
+    vagrant up
```

Comment:
This requires the user to have the Vagrantfile.
The instructions in this document do not assume that the user is in a clone of the repository.

Instead, instruct that the user downloads the Vagrantfile.
I'm not sure which tools are available on Windows by default, but for the release process we do:

```
curl https://raw.githubusercontent.com/mesosphere/dcos-e2e/master/admin/release.sh
```

Reply:
The download is two bullet points up.

Reply:
My mistake - sorry!